### PR TITLE
Wip/hughsie/fixes

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

**Comment by superm1** [comment]
Lgtm, but I wonder if the help string for force might need expansion?

**Comment by hughsie** [comment]
Something like:

    -                       _("Override plugin warning"), NULL },
    +                       _("Override warnings and force the action"), NULL },

File: src/meson.build [hunk 1]
```diff
@@ -40,6 +40,7 @@ libfwupdprivate = static_library(
     'fu-progressbar.c',
     'fu-quirks.c',
     'fu-smbios.c',
+    'fu-systemd.c',
```
Comment (superm1) [inline src/meson.build]: Given these symbols are only used with a systemd enabled compile, shouldn't this file be dynamic?

File: src/fu-self-test.c [hunk 2]
```diff
@@ -3677,6 +3677,7 @@ main (int argc, char **argv)
 	g_setenv ("FWUPD_SYSCONFDIR", TESTDATADIR_SRC, TRUE);
 	g_setenv ("FWUPD_SYSFSFWDIR", TESTDATADIR_SRC, TRUE);
 	g_setenv ("FWUPD_LOCALSTATEDIR", "/tmp/fwupd-self-test/var", TRUE);
+	g_setenv ("FWUPD_IGNORE_SYSTEMD", "1", FALSE);
```
Comment (superm1) [inline src/fu-self-test.c]: This isn't actually used

File: plugins/dell/fu-self-test.c [hunk 3]
```diff
@@ -485,6 +485,7 @@ main (int argc, char **argv)
 	sysfsdir = fu_common_get_path (FU_PATH_KIND_SYSFSDIR_FW);
 	g_setenv ("FWUPD_UEFI_ESP_PATH", sysfsdir, TRUE);
 	g_setenv ("FWUPD_DELL_FAKE_SMBIOS", "1", FALSE);
+	g_setenv ("FWUPD_IGNORE_SYSTEMD", "1", FALSE);
```
Comment (superm1) [inline plugins/dell/fu-self-test.c]: Not actually used

**Comment by superm1** [comment]
> Something like:

Yeah

File: src/meson.build (hunk 1 above)
Comment (hughsie) [reply src/meson.build]: Good idea, fixed.

File: plugins/dell/fu-self-test.c (hunk 3 above)
Comment (hughsie) [reply plugins/dell/fu-self-test.c]: Agree, that's what I'm using in another branch. I've removed it here.

File: src/fu-self-test.c (hunk 2 above)
Comment (hughsie) [reply src/fu-self-test.c]: Fixed.

**Pull request merged**